pingrid/impl.py

```diff
@@ -234,14 +286,14 @@ def _tile(da, tx, ty, tz, clipping):
     return im
 
 
-def empty_tile(width: int = 256, height: int = 256):
+def empty_tile(width: int = 256, height: int = 256, ncolors: int = 256):
     # If tile size were hard-coded, this could be a constant instead
     # of a function, but we're keeping open the option of changing
     # tile size. Also, numpy arrays are mutable, and having a mutable
     # global constant could lead to tricky bugs.
     im = apply_colormap(
         np.full([height, width], np.nan),
-        np.zeros((256, 4)),
+        np.zeros((ncolors, 4)),
```

Comment:
I don't think this is correct. `apply_colormap` is specific to opencv, so it requires a 256-element colormap.

We should eventually modify apply_colormap so that it takes a ColorScale instead of an opencv colormap; but when we do that, this function still won't need an `ncolors` parameter. The colormap resolution makes no difference here; all we need is a colormap that maps all values to (0, 0, 0, 0). We can just pick an arbitrary ncolors, like 2, and hard-code it.